src/pages/documentation/adobe-analytics/index.md

```diff
@@ -254,15 +230,7 @@ To update the SDK configuration programmatically, use the following information
 
 ## Update Analytics configuration
```

Comment:
Let's also remove configuration keys + Update Analytics configuration section - same for all index.md